pom.xml

```diff
@@ -100,6 +122,7 @@
     <maven.resources.plugin.version>3.0.1</maven.resources.plugin.version>
     <maven.source.plugin.version>2.4</maven.source.plugin.version>
     <maven.surefire.plugin.version>2.19.1</maven.surefire.plugin.version>
+    <maven.dependency.plugin.version>3.0.0-SNAPSHOT</maven.dependency.plugin.version>
```

Comment:
For circleci to speed up caching dependencies between builds `mvn dependency:go-offline` happens in dependency stage before cache is created for next build, but before actual build is run (don't want to cache build artifacts from this build).